modules/agar/src/main/ts/ephox/agar/api/TestLogs.ts

```diff
@@ -16,49 +16,45 @@ export interface TestLogs {
 const DISABLE_LOGGING = false;
 
 // Pop level needs to change the parent. This would be so much easier with zippers.
-const modifyStartedEntryTo = (entries: TestLogEntry[], f): TestLogEntry[] => {
-  return Arr.last(entries).fold(
-    () => entries,
-    (lastEntry) => {
-      // If the last entry has started, and has entries,
-      if (lastEntry.state === TestLogEntryState.Started) {
-        return Arr.last(lastEntry.entries).fold(
-          () => {
-            // We have no entries, so just modify us
+const modifyStartedEntryTo = (entries: TestLogEntry[], f): TestLogEntry[] => Arr.last(entries).fold(
+  () => entries,
+  (lastEntry) => {
+    // If the last entry has started, and has entries,
+    if (lastEntry.state === TestLogEntryState.Started) {
+      return Arr.last(lastEntry.entries).fold(
+        () =>
+        // We have no entries, so just modify us
+          entries.slice(0, entries.length - 1).concat([ f(lastEntry) ])
```

Comment:
This is a bit weird, I'm thinking the comment should be above and just have this be a single line? eg:

```
// We have no entries, so just modify us
() => entries.slice(0, entries.length - 1).concat([ f(lastEntry) ]),
// Great name!
(lastEntryLastEntry) => {
```

Reply:
I moved the comment.